### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#38]
```
Here is my proposal of a German translation. Comments welcome :-)

```

Original issue reported on code.google.com by `fschm...@gmail.com` on 7 Jan 2011 at 4:26

Attachments:
- [german_translation.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-38/comment-0/german_translation.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, I'll integrate this after my current feature
```

Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 12:10
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 12:40
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 12:42
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
a few improvements for the german translation
```

Original comment by `fschm...@gmail.com` on 8 Jan 2011 at 6:34

Attachments:
- [german-strings-update.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-38/comment-4/german-strings-update.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.7
```

Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 11:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
